src/dmd/mars.d

```diff
@@ -2303,6 +2303,27 @@ bool parseCommandLine(const ref Strings arguments, const size_t argc, ref Param
             else
                 goto Lerror;
         }
+        debug(LOG)
```

Comment:
I would leave this in release binaries but issue an error that logging was no compiled into the binary